creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -68,37 +79,38 @@ const updateReplicaSetJobProcessor = async function ({
   let healthyNodes = []
   healthyNodes = await getCachedHealthyNodes()
   if (healthyNodes.length === 0) {
-    audiusLibs = await initAudiusLibs({
-      enableEthContracts: true,
-      enableContracts: true,
-      enableDiscovery: false,
-      enableIdentity: true,
-      logger
-    })
-    const { services: healthyServicesMap } =
-      await audiusLibs.ServiceProvider.autoSelectCreatorNodes({
-        performSyncCheck: false,
-        whitelist: reconfigNodeWhitelist,
-        log: true
+    try {
+      audiusLibs = await initAudiusLibs({
+        enableEthContracts: true,
+        enableContracts: true,
+        enableDiscovery: false,
+        enableIdentity: true,
+        logger
       })
-    healthyNodes = Object.keys(healthyServicesMap || {})
-    if (healthyNodes.length === 0)
-      throw new Error(
-        'Auto-selecting Content Nodes returned an empty list of healthy nodes.'
-      )
-    await cacheHealthyNodes(healthyNodes)
+      const { services: healthyServicesMap } =
+        await audiusLibs.ServiceProvider.autoSelectCreatorNodes({
+          performSyncCheck: false,
+          whitelist: reconfigNodeWhitelist,
+          log: true
+        })
+      healthyNodes = Object.keys(healthyServicesMap || {})
+      if (healthyNodes.length === 0)
+        throw new Error(
+          'Auto-selecting Content Nodes returned an empty list of healthy nodes.'
+        )
+      await cacheHealthyNodes(healthyNodes)
+    } catch (e: any) {
+      const errorMsg = `Error initting libs and auto-selecting creator nodes: ${e.message}: ${e.stack}`
+      logger.error(errorMsg)
+      return {
+        errorMsg,
```

Comment:
you think we should return `errorMsg` that includes the stack as well? i figured maybe better to just log it above but return without it - this is a nit your call

Reply:
eh shouldn't matter much imo. the rest of the return value is short anyway so the extra stack might help avoid needing to search loggly

Reply:
ya cool